canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.scss

```diff
@@ -22,6 +22,15 @@
 		margin-bottom: $spacing-03;
 	}
 	.properties-checkboxset-container {
+		.wrapper{
+			display: flex;
+			justify-content:space-between;
+			width: 90px;
+			.tooltip-container{
+				flex:1;
+				margin-top: 8px;
```

Comment:
spacing token.  This seems odd as well.  Why is this needed?

Reply:
Removed it. used it to push down the icons & make them align with checkbox, instead changed the .tooltip-container class 
<img width="313" alt="image" src="https://user-images.githubusercontent.com/89609176/174508538-343b16b4-85e2-415d-8250-24a359a65497.png">